test/integration/data_repo_client_test.rb

```diff
@@ -144,68 +144,110 @@ def skip_if_api_down
     skip_if_api_down
     snapshot = @data_repo_client.get_snapshot(@snapshot_id)
     assert snapshot.present?
-    expected_keys = %w(id name description createdDate source tables
-                       relationships profileId dataProject accessInformation).sort
+    expected_keys = %w[id name description createdDate source tables
+                       relationships profileId dataProject accessInformation].sort
     assert_equal expected_keys, snapshot.keys.sort
   end
 
   test 'should get file in snapshot' do
     skip_if_api_down
     file_info = @data_repo_client.get_snapshot_file_info(@snapshot_id, @file_id)
     assert file_info.present?
-    assert_equal @file_id, file_info.dig('fileId')
-    assert file_info.dig('size') > 0, 'did not get a valid file size'
+    assert_equal @file_id, file_info['fileId']
+    assert file_info['size'].positive?, 'did not get a valid file size'
```

Comment:
I find `> 0` easier to understand than `.positive?`.  The latter is idiomatic Ruby, but also _uniquely_ Ruby.

Reply:
That was a `rubocop` suggestion which was why I did it.  I tend to agree, but I also wanted to start getting in the practice of accepting the linter's advice...

Reply:
Given [Rubocop's default here](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/NumericPredicate) impedes readability for both of us, we could change the linting rule by adding:

```
Style/NumericPredicate
  Enabled: false
```

in `rubocop.yml`.  (We occasionally change defaults in our JS lint rules, via `.eslintrc.js`, as well.)